### PR TITLE
RE-1162 Add lint jobs for rpc-upgrades

### PR DESCRIPTION
**Pull request by mattt416**
This PR adds jobs which will replace rpc-upgrades' Travis CI usage.

Issue: [RE-1162](https://rpc-openstack.atlassian.net/browse/RE-1162)

### CODE REVIEW

**Comment by mattt416** [comment]
recheck_xenial_lint_test

**Comment by mattt416** [comment]
recheck_cit_unit_tests

**Comment by mattt416** [comment]
recheck_cit_xenial_functional_test

**Comment by mattt416** [comment]
recheck_all

**Comment by mattt416** [comment]
https://github.com/rcbops/rpc-upgrades/pull/105 should merge before this does.

**Comment by mattt416** [comment]
recheck_cit_xenial_lint_test
rechecK_cit_xenial_functional_test

**Comment by mattt416** [comment]
recheck_cit_unit_tests

File: rpc_jobs/rpc_upgrades.yml [hunk 1]
```diff
@@ -18,6 +18,28 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
+- project:
+    name: "rpc-upgrades-aio-tox-pr"
+    repo_name: "rpc-upgrades"
+    repo_url: "http://github.com/rcbops/rpc-upgrades"
```
Comment (git-harry) [inline rpc_jobs/rpc_upgrades.yml]: https?

**Comment by mattt416** [comment]
Good catch @git-harry -- was a copy pasta fail.  I've updated all http:// references in rpc_jobs/rpc_upgrades.yml.

**Pull request merged**